rules/mandatory-vs-suggested-pr-changes/rule.md

```diff
@@ -0,0 +1,57 @@
+---
+seoDescription: TODO
```

Comment:
Does this need attention before we publish?

Reply:
Sorted